docs/configuration.md

```diff
@@ -80,6 +80,8 @@ cascade_config:
 # The available options for the sarif_options keyword are:
 # 1) `include_suppressed: true/false` -This option allows users to include/exclude suppressed/excluded results 
 #    in their sarif reports. Currently this is supported for NPM audit reports
+# 2) `include_active: true/false` - This option allows users to include or exclude results from active (non enforced)
+#    scanners in their sarif reports.  Default is false.
```

Comment:
I wonder if we should rename `include_active` to `include_non_enforced`?

All scanners that run in a salus scan are active scanners.

Reply:
Our goal here is to exclude non-enforced active scanners, not active scanners.

Reply:
+1

Reply:
Sounds good will rename